app/views/source_sets/show.html.erb

```diff
@@ -24,7 +24,7 @@
         <h1><%= inline_markdown(@source_set.name) %></h1>
         <span class='subheading'>Primary Source Set</span>
         <% if @source_set.authors.present? %>
-          <span class='byline'>By <%= authors(@source_set).join(' and ') %></span>
+          <span class='byline'>By <%= @source_set.author_list.join(' and ') %></span>
```

Comment:
Do we ever expect a set to have more than 2 authors? I'm just wondering how it'll look if we we have something like `Moomin Troll, Moominvalley Historical Society and Moominmama and Moominpapa` ...